site2/docs/admin-api-brokers.md

```diff
@@ -48,6 +48,33 @@ admin.brokers().getActiveBrokers(clusterName)
 
 <!--END_DOCUSAURUS_CODE_TABS-->
 
+### Get the information of the leader broker
+
+Fetch the information of the leader broker.
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--pulsar-admin-->
+
+```shell
+$ pulsar-admin brokers leader-broker
+```
+
```

Comment:
To allow users to have more info (flags and descriptions), it is better to add the URL of the `pulsar-admin brokers leader-broker` command on [pulsar-admin](http://pulsar.apache.org/tools/pulsar-admin/2.8.0-SNAPSHOT/#brokers). For example:

`For more information, see [here](xxx).`

Reply:
Thanks for your tip.
There is no more args for this command.

Reply:
Thanks for your tip.
There is no more args for this command.